### ISSUE TITLE
When using @ConfigurationProperties without a prefix the first letter of the property is removed

### ISSUE THREAD

**Issue by katiaxyz** [rodnansol/spring-configuration-property-documenter#79]
**Describe the bug**
Wrong naming for the property when @ConfigurationProperties has empty prefix, i.e. empty group.

**To Reproduce**

Define the configuration as below:

```
@Component
@Data
@ConfigurationProperties
public class MyConfigurationProperties {
    /**
     * Generic property
     */
    private String myProperty = "propertyValue";
}

```
**Expected behavior**
The expected behaviour is to have a property named my-property as also present in spring-configuration-metadata.json and empty group in place of "+".

```
{
      "name": "my-property",
      "type": "java.lang.String",
      "description": "Generic property",
      "sourceType": "com.kat.configuration.MyConfigurationProperties",
      "defaultValue": "propertyValue"
    }
```

**Screenshots**
<img width="698" alt="Screenshot 2023-07-11 at 16 59 40" src="https://github.com/rodnansol/spring-configuration-property-documenter/assets/137886114/e3a6eb35-20bc-4a87-bd70-624781d31687">

**Additional context**
plugin configuration in pom.xml:

                    <plugin>
			<groupId>org.rodnansol</groupId>
			<artifactId>spring-configuration-property-documenter-maven-plugin</artifactId>
			<version>0.6.0</version>
			<executions>
				<execution>
					<id>generate-adoc</id>
					<phase>process-classes</phase>
					<goals>
						<goal>generate-property-document</goal>
					</goals>
					<configuration>
						<type>ADOC</type>
						<outputFile>path-to-adoc-file</outputFile>
						<failOnError>true</failOnError>
					</configuration>
				</execution>
			</executions>
		</plugin>

**Comment by nandorholozsnyak** [comment]
It won't work with the 0.6.1 version. Let me fix that, it should be easy.

**Comment by Riggs333** [comment]
@nandorholozsnyak Thanks for the quick fix!
As I'm running into the same issue with version 0.6.1, is there a plan when you will release the fix?

**Comment by nandorholozsnyak** [comment]
Hello @Riggs333

I should have already released it, give me some time to pack the release together.

**Comment by nandorholozsnyak** [comment]
Hello @Riggs333 

Sorry for the late reply, but the fix should be available in the newly released 0.7.0 version!!
